README.md

```diff
@@ -32,40 +32,14 @@ You will need either a connected Jetpack site or use the following filter to for
 
 `add_filter( 'jetpack_development_mode', '__return_true' );`
```

Comment:
Latest version of Jetpack throws tons of deprecation warnings here, saying to use `jetpack_offline_mode` instead.

Reply:
Updated readme in https://github.com/Automattic/wc-calypso-bridge/pull/614/commits/2011d46aa9d8e07e61999f3c078efbe0cece6df0